scripts/download/lima.mjs

```diff
@@ -52,15 +52,20 @@ async function getAlpineLima(arch) {
 
 export default function run() {
   let platform = os.platform();
-  let arch = 'x86_64';
+  const arch = 'x86_64';
+
+  const promises = [];
 
   if (platform === 'darwin') {
     platform = 'macos';
-    if (process.env.M1) {
-      arch = 'aarch64';
-      platform = `macos-${ arch }`;
+    if (process.env.M1 !== '0' && (process.env.M1 || process.arch === 'arm64')) {
+      const m1arch = 'aarch64';
+      const m1platform = `macos-${ m1arch }`;
+
+      promises.push([getLima(m1platform), getAlpineLima(m1arch, m1platform)]);
```

Comment:
You wan to push pomises, not array of promises:
```suggestion
      promises.push(getLima(m1platform), getAlpineLima(m1arch, m1platform));
```

Reply:
Sadly the code has been refactored (`Lima` is now a class ... in typescript).

Offhand, I suspect I want to move some of this code to scripts/postinstall.ts, but I don't have the energy now...